src/runtime_src/core/tools/common/XBMain.h

```diff
@@ -20,6 +20,7 @@
 // Include files

```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.